resources/views/layouts/user.blade.php

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
```

Comment:
Breakdown the header, and main content into smaller components. this is just code repetition